mw_jwt.go

```diff
@@ -100,31 +108,41 @@ func (k *JWTMiddleware) getSecretFromURL(url, kid, keyType string) ([]byte, erro
 	return nil, errors.New("No matching KID could be found")
 }
 
-func (k *JWTMiddleware) getIdentityFomToken(token *jwt.Token) (string, bool) {
-	// Try using a kid or sub header
+// Try using a kid or configured claim or sub claim
+func (k *JWTMiddleware) getIdentityFomToken(token *jwt.Token) (string, bool, error) {
 	idFound := false
 	var tykId string
```

Comment:
I suppose we don't need these variables any more as "pass-through" logic was removed/changed below